var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -32,11 +32,12 @@ class Relion(CMakePackage, CudaPackage):
     electron cryo-microscopy (cryo-EM)."""
 
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
-    git      = "https://github.com/3dem/relion.git"
+    url      = "https://github.com/3dem/relion"
 
-    version('develop')
-    version('2.1', tag='2.1')
-    version('2.0.3', tag='2.0.3')
+    version('2.1', git='https://github.com/3dem/relion.git', tag='2.1')
+    version('2.0.3', git='https://github.com/3dem/relion.git', tag='2.0.3')
+    version('develop', git='https://github.com/3dem/relion.git')
+    version('beta-3', git='https://bitbucket.org/scheres/relion-3.0_beta.git')
```

Comment:
Why not call the version `3.0_beta` and mark `2.1` as `preferred=True`?

Reply:
Been a while since added things to Spack... Thanks!